instrumentation/rails/Appraisals

```diff
@@ -5,9 +5,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 appraise 'rails-5.2' do
-  gem 'rails', '~> 5.2'
+  gem 'rails', '~> 5.2.0'
 end
 
 appraise 'rails-6.0' do
-  gem 'rails', '~> 6.0'
+  gem 'rails', '~> 6.0.0'
```

Comment:
It was using `6.1` instead of `6.0` which has different config support.  Which is why the tests are exploding on the repo right now.